tools/run_desktop_tests.py

```diff
@@ -127,6 +130,11 @@ def categorize_tests(self):
         return {TO_RUN:tests_to_run, SKIP:local_skiplist, NOT_FOUND:not_found, WITH_SERVER:tests_with_server}
         
 
+    def test_file_with_keys(self, test_file):
+        boost_keys = "  --report_format=xml --report_level=detailed --log_level=test_suite --log_format=xml " if self.boost_tests else ""
```

Comment:
Why do we need two spaces in the beginning of boost_keys string? One space is enough,  isn't it?